pkg/output/output.go

```diff
@@ -42,9 +43,35 @@ import (
 // that the output can be shown correctly.
 var emptyBuffer = []byte{91, 10, 32, 32, 10, 93}
 
+var hideEmptyFields bool
+
+// AddHideEmptyFieldsFlag adds the --hide-empty-fields flag to a command
+func AddHideEmptyFieldsFlag(fs *pflag.FlagSet) {
+	fs.BoolVar(
+		&hideEmptyFields,
+		"hide-empty-fields",
+		false,
+		"Hide fields that do not contain any data",
+	)
+}
```

Comment:
From the ticket:

```
A one-letter or two-letter flag would be appropriate for this, and visible as an option in the help messages. The flag for this might be -cc (column collapse?) or -hc (hide columns?).
```